### DISCUSSION TITLE
InertiaJS v2.0 Support

### DISCUSSION THREAD

**Discussion by FahrenheitIndi** [RobertBoes/inertia-breadcrumbs#23] [Q&A] [answered]
Could you please update support for InertiaJS v2.0?

**Comment by RobertBoes** [comment, marked as answer]
There already is a pre-release version for 2.0: https://github.com/RobertBoes/inertia-breadcrumbs/releases/tag/0.7.0-beta

I just haven't had the time to test it, as I had too many issues with the Inertia 2.0 beta. Feel free to test it and provide feedback
